jobQueue/jobWrapper.js

```diff
@@ -103,6 +105,8 @@ function processJobOnComplete(job, logObject) {
 
         /* The second argument should match the activity type in
          /config/activityLog.js */
+        queueTimeActivityLogs
+          .update(jobResultObj.recordCount, jobResultObj.queueTime);
         activityLogUtil.printActivityLogString(logObject, 'worker');
       }
```

Comment:
Looks line 106 to 107 were met for the function on line 110. So you might want to do some swapping here.